config/prow/grafana-dashboard/influxdb_pv.yaml

```diff
@@ -6,8 +6,9 @@ metadata:
   name: influxdb
   namespace: grafana-dashboard
 spec:
+  storageClassName: ibmc-vpc-block-retain-10iops-tier
```

Comment:
why not default SC? is it because to retain the volume?

Reply:
yes, this is for retaining the volume